themes/aframe/layout/blog.ejs

```diff
@@ -8,27 +8,56 @@
 <div class="content content--blog copy post">
   <div class="copy__wrap">
     <% if (page.blog_index) { %>
-
       <h1 class="page__title"><%- config.blog_title %></h1>
 
       <%- page.content %>
 
       <% site.posts.sort('date', -1).each(function (post) { %>
-        <article class="post">
-          <h2 class="post__heading"><%- post.title %></h2>
-          <h4 class="post__timestamp"><%- post.date.format('MMM D[,] YYYY') %></h4>
-          <div class="post__excerpt"><%- post.excerpt %></div>
+        <article class="post post-listing">
+          <a href="<%- page_url(post.path) %>">
+            <h2 class="post__heading"><%- post.title %></h2>
+          </a>
+          <%- partial('partials/blog/metadata', {post: post}) %>
+
+          <% if (post.awoa) { %>
+            <p>What's up with A-Frame for the week of <%- blog_date_subtract_week(post.date) %> to <%- blog_date(post.date) %>.</p>
+            <%- partial('partials/blog/awoa_projects_showcase', {
+              projects: post.awoa.projects
+            }) %>
+          <% } %>
+
+          <% if (!post.awoa) { %>
+            <%- partial('partials/blog/image', {image: post.image, title: post.title}) %>
+            <div class="post__excerpt"><%- post.excerpt %></div>
+          <% } %>
+
           <a href="<%- page_url(post.path) %>" class="post__more">Read more &hellip;</a>
         </article>
       <% }) %>
 
     <% } else { %>
-
-      <article class="post">
+      <article class="post post-full">
         <h1 class="hidden-visually"><%- config.blog_title %></h1>
         <h2 class="post__heading"><%- page.title %></h2>
-        <h4 class="post__timestamp"><%- page.date.format('MMM D[,] YYYY') %></h4>
+        <%- partial('partials/blog/metadata', {post: page}) %>
+
+        <% if (page.awoa) { %>
+          <%- partial('partials/blog/awoa_projects_showcase', {
+            projects: page.awoa.projects
+          }) %>
+        <% } %>
+
+        <%- partial('partials/blog/image', {image: page.image, title: page.title}) %>
+
         <%- page.content %>
+
+        <% if (page.awoa) { %>
+          <%- partial('partials/blog/awoa_projects_text', {projects: page.awoa.projects}) %>
+        <% } %>
+
+        <% if (page.author) { %>
+          <p class="post__author_footer">By <%- blog_attribution(page.author) %></p>
```

Comment:
hmm, it's a bit odd to have two bylines:

![image](https://cloud.githubusercontent.com/assets/203725/14481434/c51b34c0-00e7-11e6-9772-8cbe5861f6af.png)

originally, we were going to do it this way:

<hr>

__WebVR Solar System__
by Chris Van
Jan 21, 2016

> Designer José Manuel is working on an animated interactive representation of our solar system.

This solar system is a little project that will grow step by step to make a big project. A solar system, following the A-Frame instructions of animation. I hope you like it. :D

By [José Manuel](https://twitter.com/chetemele), graphic designer

<hr>

but obviously that's a bit confusing. I know we don't have to over-engineer it (and the authors aren't showing up in [the feed](http://localhost:4000/feed.xml) anyway).

your call. but personally I found it odd to read a timestamp and then `@chetemele`. I know Twitter, and I know the `@` convention, but even I was a bit confused (a) that that was an author name and (b) that that was a link to Twitter.

I'd simply recommend adding `by ` before the author name